### PR TITLE
create navigation

### LINKED ISSUE #5: Create navigation

### PR DESCRIPTION
**Pull request by BogdanCDima**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,37 @@
...
+  Switch,
+  Route,
+} from "react-router-dom";
 
 function App() {
 
   return (
     <div className="App">
       <Header />
+      <Router>
```
Comment (alexandra-iacob) [inline src/App.js]: the design is with tabs, not with different pages. when you change from today to 7days the url should not change

File: src/components/Header/index.jsx [hunk 2]
```diff
@@ -50,7 +50,7 @@ export default function Header() {
     useEffect(() => {
         curentLocation && dispatch(getCurrentWeatherOnCurrentLocation(curentLocation, unit));
         // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (alexandra-iacob) [inline src/components/Header/index.jsx]: do you still need this disabled?

Comment (BogdanCDima) [reply src/components/Header/index.jsx]: no, removed it!

File: src/App.js (hunk 1 above)
Comment (BogdanCDima) [reply src/App.js]: fixed that.

**Pull request merged**